src/components/App.js

```diff
@@ -57,6 +58,10 @@ const App = () => {
 
   return (
     <div className="App">
+      <Header
+        TOTAL_PHOTO_COUNT={TOTAL_PHOTO_COUNT}
```

Comment:
The prop name should probably not be snake_cased and uppercased, juse `totalPhotoCount`